src/esp/sim/Simulator.cpp

```diff
@@ -606,8 +614,12 @@ bool Simulator::recomputeNavMesh(nav::PathFinder& pathfinder,
                  "loaded without renderer initialization.",
                  false);
 
-  assets::MeshData::uptr joinedMesh =
-      resourceManager_->createJoinedCollisionMesh(config_.scene.id);
+  assets::MeshData::uptr joinedMesh = assets::MeshData::create_unique();
```

Comment:
Make sure there is some initial value here (e.g., nullptr).

Reply:
The new object is the initial value. More verts/indices are added by STATIC non-stage objects.